scripts/install

```diff
@@ -99,8 +98,13 @@ ynh_psql_setup_db --db_user=$db_user --db_name=$db_name
 ynh_script_progression --message="Setting up source files..." --weight=3
 
 ynh_app_setting_set --app=$app --key=final_path --value=$final_path
-# Download, check integrity, uncompress and patch the source from app.src
-ynh_setup_source --dest_dir="$final_path"
+# Note: We use git instead of ynh_setup_source, cause this repo use submodules
```

Comment:
if you really need to switch to git clone, you need to fix the commit to use + you should remove app.src